plugins/linux-capture/pipewire.c

```diff
@@ -610,16 +663,16 @@ static void on_core_error_cb(void *user_data, uint32_t id, int seq, int res,
 
 static void on_core_done_cb(void *user_data, uint32_t id, int seq)
 {
-	UNUSED_PARAMETER(seq);
-
 	obs_pipewire_data *obs_pw = user_data;
 
-	if (id == PW_ID_CORE)
+	if (id == PW_ID_CORE && (obs_pw->server_version_sync == 0 ||
+				 obs_pw->server_version_sync == seq))
```

Comment:
I think this can be simplified to:

```suggestion
	if (id == PW_ID_CORE && obs_pw->server_version_sync == seq)
```

(This pre-existing code is actually useless. I've added it a few years ago, together with a call to `pw_thread_loop_wait()` in a different part of the file, but after some scrutiny I removed the `pw_thread_loop_wait()` call, but seems like I forgot the corresponsing `pw_thread_loop_wait()`s)